test/pdfs/.gitignore

```
@@ -696,3 +696,5 @@
 !issue18911.pdf
 !issue19207.pdf
 !issue19239.pdf
+!link.pdf
+!mozilla.org.pdf
```

Can we please have better named test files, e.g. as follows?
```suggestion
!bug1019475_1.pdf
!bug1019475_2.pdf
```